terraform/cluster/variables.tf

```diff
@@ -44,7 +44,7 @@ variable "pvt_key" {
 
 variable "instance_size" {
   description = "The instance size to use"
-  default = "1gb"
+  default = "s-1vcpu-1gb"
```

Comment:
Interestingly, `1gb` worked just fine and actually created 1gb droplets for me 🤔 Not sure about this. Will check the DO-terraform documentation.

Reply:
OK, while 1gb seems to work it seems that this is outdated and the above is the correct "slug" for the size:
 - https://registry.terraform.io/providers/digitalocean/digitalocean/latest/docs/resources/droplet#size
 - https://developers.digitalocean.com/documentation/v2/#list-all-sizes


Thanks again Evan 👍🏼